src/Dom/Document.php

```diff
@@ -536,6 +537,35 @@ private function move_invalid_head_nodes_to_body() {
 		}
 	}
 
+	/**
+	 * Converts a possible head[profile] attribute to link[rel=profile].
+	 *
+	 * The head[profile] attribute is only valid in HTML4, not HTML5.
+	 * So if it exists and there's no link[rel=profile], add it to the <head> as a link[rel=profile].
+	 */
+	private function convert_head_profile_to_link() {
+		$profile = $this->head->getAttribute( 'profile' );
+		if ( ! $profile ) {
```

Comment:
This should handle the case of someone putting `profile=""` perhaps, so:

```suggestion
		if ( ! $this->head->hasAttribute( 'profile' ) ) {
```

Reply:
If `<head profile="">`, do you think we should strip the `profile` but not create a `<link re="profile">` ?

Reply:
Sorry, you just answered that in https://github.com/ampproject/amp-wp/pull/4193#discussion_r372636143

Reply:
Applied in https://github.com/ampproject/amp-wp/pull/4193/commits/6ebae07a5a2f804b73eb465120e41f06259ec2bd